protobuf-format.md

```diff
@@ -0,0 +1,133 @@
+# Protocol Buffers Event Format for CloudEvents  - Version 0.1
+
+## Abstract
+
+The Protocol Buffers Format for CloudEvents (CE) defines the encoding
+of CloudEvents in the Protocol Buffers binary format.
+
+## Status of this document
+
+This document is a working draft.
+
+## 1. Introduction
+
+This specification defines how the [Context
+Attributes](spec.md#context-attributes) Attributes defined in the
+CloudEvents specification MUST be encoded in the protocol buffer
+binary format. Transcoding to and from other formats (e.g. JSON) is
+out of the scope of this document.
+
+Protocol Buffers are a language-neutral, platform-neutral extensible
+mechanism for serializing structured data. The [Google reference
+implementation of Protocol
+Buffers](https://github.com/protocolbuffers/protobuf) includes support
+for an interface descriptor language (IDL), and this document makes
+use of language level 3 IDL from Protocol Buffers v3.5.0. CloudEvents
+systems using Protocol Buffers are not required to use the IDL or any
```

Comment:
s/required/manadated/ to avoid the RFC checker... which reminds me, can you modify the Makefile as part of this PR so that the RFC checker is run on this doc too?

Reply:
Done